core/server/data/migrations/versions/2.28/2-add-db-backup-content-permission-to-roles.js

```diff
@@ -1,46 +1,15 @@
-const logging = require('../../../../../shared/logging');
-const utils = require('../../../schema/fixtures/utils');
-
-const relationFixtures = {
-    from: {
-        model: 'Role',
-        match: 'name',
-        relation: 'permissions'
-    },
-    to: {
-        model: 'Permission',
-        match: ['object_type', 'action_type']
-    },
-    entries: {
-        Administrator: {
-            db: 'backupContent'
-        },
-        'DB Backup Integration': {
-            db: 'backupContent'
-        }
-    }
-};
-
-module.exports = {
-    config: {
-        transaction: true
-    },
-
-    async up(options) {
-        try {
-            await utils.addFixturesForRelation(relationFixtures, options);
-            return logging.info('Completed adding db.backupContent permission to roles');
-        } catch (err) {
-            return logging.warn('Issue adding db.backupContent permission to roles');
-        }
-    },
-
-    async down(options) {
-        try {
-            await utils.removeFixturesForRelation(relationFixtures, options);
-            return logging.info('Completed removing db.backupContent permission from roles');
-        } catch (err) {
-            return logging.warn('Issue removing db.backupContent permission from roles');
-        }
-    }
-};
+const {
+    addPermissionToRole,
+    combineTransactionalMigrations
+} = require('../../utils');
+
+module.exports = combineTransactionalMigrations(
+    addPermissionToRole({
```

Comment:
Annoyingly this is more verbose than `addPermissionWithRoles` - consider adding an `addPermissionToRoles` function? :thinking:

Reply:
It's worth noting that the migration utils are (at least designed to be) idempotent, so we could have just as easily used `addPermissionWithRoles` here and explicitly defined the permission - but it's like this for two reasons:

1. I like to keep the functionality as close to what the old migrations are
2. I think it's better to be specific and explicit in what we want to perform